src/pywrangler/pyspark/wranglers/interval_identifier.py

```diff
@@ -163,7 +163,12 @@ def _drop_duplicated_marker(self, marker_column: Column, window: Window,
         mask_only = denoised == marker
 
         # use shifted column to identify subsequent duplicates
-        shifted = F.lag(denoised, count=count_lag).over(window)
+        try:
+            # keep this code for backwards compatibility
+            shifted = F.lag(denoised, count=count_lag).over(window)
+        except TypeError:
+            # signature of the function changed in PySpark 3.0.0
+            shifted = F.lag(denoised, offset=count_lag).over(window)
```

Comment:
There was another call to the `lag` function that used the wrong parameters, which I've now put inside of a try / except block to ensure backwards compatibility.